frontend/src/objects/Fleet.ts

```diff
@@ -148,14 +150,14 @@ export class Fleet {
   runCountDown() {
     if (!this.socket) return;
     const message: WebSocketMessage = {
-      data: UserStore.player.countDown,
+      data: null,
       messageType: WebSocketMessageType.START_COUNTDOWN,
     };
```

Comment:
isn't the countdown seconds number sent by the server ? if not why is it in a variable up there